apps/nextjs/src/components/AppComponents/Chat/chat-message/index.tsx

```diff
@@ -190,6 +190,9 @@ function MessageWrapper({
   children,
   type,
 }: MessageWrapperProps) {
+  const testId = errorType
```

Comment:
Should this be === ?

Reply:
The behaviour I'm looking for is: If there's an error type, use `chat-message-wrapper-error-{errorType}`. Otherwise use `chat-message-wrapper-{type}`